package.exs

```diff
@@ -3,16 +3,16 @@ defmodule GenRPC.Mixfile do
 
```

Comment:
Concerning `package.exs` - we should just drop it.  It's an old way of publishing Erlang libraries to Hex, requiring Mix.

But I'll leave it up to you whether to handle this now or later.

Reply:
I'll do it now.